packages/autometrics/src/wrappers.ts

```diff
@@ -278,26 +278,14 @@ export function autometrics<F extends FunctionSig>(
       const autometricsDuration = (performance.now() - autometricsStart) / 1000;
 
       counter.add(1, {
-        [FUNCTION_LABEL]: functionName,
-        [MODULE_LABEL]: moduleName,
+        ...commonCounterAttributes,
         [RESULT_LABEL]: "ok",
         [CALLER_FUNCTION_LABEL]: callerFunction,
         [CALLER_MODULE_LABEL]: callerModule,
-        ...counterObjectiveAttributes,
       });
 
-      histogram.record(autometricsDuration, {
-        function: functionName,
-        module: moduleName,
-        caller_function: callerFunction,
-        caller_module: callerModule,
-        ...histogramObjectiveAttributes,
-      });
-
-      concurrencyGauge?.add(-1, {
-        function: functionName,
-        module: moduleName,
-      });
```

Comment:
I noticed we sometimes submitted too many labels, so I refactored these objects a little bit so they're more consistent.